src/components/NcColorPicker/NcColorPicker.vue

```diff
@@ -166,6 +204,12 @@ const props = withDefaults(defineProps<{
 	 */
 	advancedFields?: boolean
 
+	/**
+	 * Allow to clear the current color.
+	 * When set the `update:modelValue` and `submit` event might emit also `undefined`.
+	 */
+	clear?: boolean
```

Comment:
We have props like this in several components, I'd expect it to match
```suggestion
	clearable?: boolean
```

Reply:
Yes makes sense! Fully support this!